### ISSUE TITLE
Jetengine - Make slugs names fields visible admin only OR insert into a wrap

### ISSUE THREAD

**Issue by siddgripp** [CrocoBlock/suggestions#2258]
Please, make slugs fields visible only to admin and appearance editor.

OR

Just insert inside an html tag so that we can hide it with css rules.

**This should appear only to those interested**
https://www.awesomescreenshot.com/image/5764614/e2f5ffab0c869dc1cabfc7c8255c1823

**Wrap text note and slug name**
https://www.awesomescreenshot.com/image/5764615/63a5632b7f9c1cfebb8650576ea48323

**How is it**
```
<div class="cx-control__info">
<h4 class="cx-ui-kit__title cx-control__title" role="banner">Label Fields</h4>
<div class="cx-ui-kit__description cx-control__description" role="note">
     Note for this field
     <br>
     Name: slug_this_field
</div>
</div>

```
**How could it be**
```
<div class="cx-control__info">
<h4 class="cx-ui-kit__title cx-control__title" role="banner">Label Fields</h4>
<div class="cx-ui-kit__description cx-control__description" role="note">
      <p>Note for this field</p> 
     <span>Name: slug_this_field</span> 
</div>
</div>

```

**Comment by siddgripp** [comment]
@Floydtm @kaskad88 @MakhonkoDenis @MjHead Hello guys! I congratulate the skillful developers and the entire Crocoblock team for a year of great achievements towards creating an amazing product with every update.

Even with all your priorities, I figured you were going to be able to add a <span> to this snippet of code.

I have not found selectors in CSS capable of pointing to this last line of text within the div. I also tried to find a solution in JS and I couldn't get a JS code that applies the <span> tag to all lines of text inside this div, even if only on the last line, as it would be enough for the selection by CSS ( div span:last) or (div br+span).
